development/README.md

```diff
@@ -282,4 +282,191 @@ $BUILD_SCRIPTS/qgis-dev-install.sh $BUILD_DIR
 
 ## Configure/build/install QGIS in Qt Creator.app
 
-TODO
+### Install QtCreator 4.0 or better
```

Comment:
Yeah @Gustry you will recognize that it is more or less the same process from my QGIS 2 notes but everything here assumes QGIS 3.x will be built.